client/clientservice/src/request_service.cpp

```diff
@@ -46,21 +46,33 @@ Status RequestServiceImpl::Send(ServerContext* context, const Request* proto_req
     if (not std::holds_alternative<bft::client::Reply>(send_result)) {
       LOG_INFO(logger_, "Send returned error");
       switch (std::get<uint32_t>(send_result)) {
-        case (concord_client_pool::Overloaded):
+        case (static_cast<uint32_t>(bftEngine::OperationResult::OVERLOADED)):
           status.set_value(grpc::Status(grpc::StatusCode::RESOURCE_EXHAUSTED, "All clients occupied"));
           break;
-        case (concord_client_pool::InvalidArgument):
+        case (static_cast<uint32_t>(bftEngine::OperationResult::INVALID_REQUEST)):
           status.set_value(grpc::Status(grpc::StatusCode::INVALID_ARGUMENT, "Invalid argument"));
           break;
-        case (concord_client_pool::TimedOut):
+        case (static_cast<uint32_t>(bftEngine::OperationResult::TIMEOUT)):
           status.set_value(grpc::Status(grpc::StatusCode::DEADLINE_EXCEEDED, "Timeout"));
           break;
-        case (concord_client_pool::ClientUnavailable):
+        case (static_cast<uint32_t>(bftEngine::OperationResult::NOT_READY)):
           status.set_value(grpc::Status(grpc::StatusCode::UNAVAILABLE, "No clients connected to the replicas"));
           break;
-        default:
+        case (static_cast<uint32_t>(bftEngine::OperationResult::INTERNAL_ERROR)):
           status.set_value(grpc::Status(grpc::StatusCode::INTERNAL, "Internal error"));
           break;
+        case (static_cast<uint32_t>(bftEngine::OperationResult::EXEC_DATA_TOO_LARGE)):
+          status.set_value(grpc::Status(grpc::StatusCode::RESOURCE_EXHAUSTED, "Execution data too large"));
+          break;
+        case (static_cast<uint32_t>(bftEngine::OperationResult::EXEC_DATA_EMPTY)):
+          status.set_value(grpc::Status(grpc::StatusCode::INTERNAL, "Execution data is empty"));
+          break;
+        case (static_cast<uint32_t>(bftEngine::OperationResult::CONFLICT_DETECTED)):
+          status.set_value(grpc::Status(grpc::StatusCode::ABORTED, "Aborted"));
```

Comment:
That error code doesn't exist in request.proto please choose a different one or change the documentation with a recommendation for the caller what to do.

Reply:
As discussed we will keep aborted for conflict_detected.